docs/faqGeneral.md

```diff
@@ -8,35 +8,43 @@ fabric8 uses the [Apache 2.0 License](http://www.apache.org/licenses/LICENSE-2.0
 
 Fabric8 is an integrated open source [DevOps](fabric8DevOps.html) and [Integration Platform](ipaas.html) which works out of the box on any [Kubernetes](http://kubernetes.io/) or [OpenShift](http://www.openshift.org/) environment and provides [Continuous Delivery](cdelivery.html), [Management](management.html), [ChatOps](chat.html) and a [Chaos Monkey](chaosMonkey.html).
 
-
-
 #### What does fabric8 do?
 
 Fabric8 (pronounced _fabricate_) gives you out of the box services that assist you when building Microservices, monoliths or any application in a linux container (Docker/Rocket) environment and is built on top of [Kubernetes](http://kubernetes.io/).
 
+#### What is Kubernetes?
+
+[Kubernetes](http://kubernetes.io) provides a [Docker](http://docker.io/) based _Container As A Service_ (orchestrates docker containers)
+
+#### How does Fabric8 relate to other open source projects?
+
+Fabric8 provides upstream innovation on concepts and features, once validated, make their way into other open source projects such as [OpenShift Origin](https://github.com/openshift/origin), [Kubernetes](http://kubernetes.io) and [Jenkins](https://jenkins.io/) (as examples). Fabric8 contributors work in the various other open source communities to align a unified developer experience.
+
 #### What value does fabric8 add over OpenShift?
 
```

Comment:
Yes, I guess I was tripped up as some of the things listed, though maybe implemented with different technologies or a different way, were listed in fabric8 section as new features.

Would it make sense to have another section, like I did for Kubernetes and you did for fabric8, to say what OpenShift adds to kubernetes? I get many questions where the lines are between kube/openshift and fabric8. So establishing kube, kube+openshift, and then fabric8 might be helpful in your faq